feature/flags.go

```diff
@@ -47,8 +47,12 @@ const OldPresence = "old-presence"
 // Mongo-based lease store, rather than by the Raft FSM.
 const LegacyLeases = "legacy-leases"
 
-// Generations will allow for model generation functionality to be used.
-const Generations = "generations"
+// Branches will allow for model branches functionality to be used.
+// Historically branches feature was called generations.
+const Branches = "branches"
+
+// OldBranchesName is the old flag name used for generations feature
+const OldBranchesName = "generations"
```

Comment:
There is no need to rename this; it obscures history. Just leave it as Generations.
```go
// Branches will allow for model branches functionality to be used.
const Branches = "branches"

// Generations will allow for model generation functionality to be used.
// This is a deprecated flag name and is synonymous with "branches" above.
const Generations = "generations"
```